### ISSUE TITLE
Feeds don't update, need to disable, restart, then re-enable to update. 

### ISSUE THREAD

**Issue by jeremiah-rs** [brief-rss/brief#461]
I'm not sure if this is a similar bug to the ones already reported on here so I am submitting this bug report. 

When I have Brief enabled, it will work and do it's thing well until I close the browser. When the browser is restarted, it stops updating the feeds. To fix this, I have to disable brief, exit firefox, then re-open Firefox and then enable Brief. Once this is done, Brief starts working again. 

Is anyone else having a similar issue?

**Comment by tanriol** [comment]
I've never seen this kind of problems.

What's your Firefox version and Brief version?
When Brief does not update feeds, does the Brief tab open? Can Brief's icon display the unread number?

Can you open `about:debugging`, choose Debug for Brief, tell Brief to update feeds and check whether there are any errors in the console?

**Comment by Sxderp** [comment]
There have been pretty big reliability and usability upgrades to breakpoints in recent Firefox versions. You should be able to easily trace the background page initialization by opening the debugger, putting down a breakpoint, and enabling / disabling in about:addons.

**Comment by jeremiah-rs** [comment]
> 
> 
> I've never seen this kind of problems.
> 
> What's your Firefox version and Brief version?
> When Brief does not update feeds, does the Brief tab open? Can Brief's icon display the unread number?
> 
> Can you open `about:debugging`, choose Debug for Brief, tell Brief to update feeds and check whether there are any errors in the console?

I am using Firefox v67.0.1 and Brief v2.6.0b2. 

The Brief tab does open, it just does nothing when I hit update and doesn't update. I have the number update turned off, I will toggle that setting and see if that improves anything. 

I will run the debug and post what I see in an edit. 

Thanks for getting back to me!

**Comment by jeremiah-rs** [comment]
> 
> 
> I've never seen this kind of problems.
> 
> What's your Firefox version and Brief version?
> When Brief does not update feeds, does the Brief tab open? Can Brief's icon display the unread number?
> 
> Can you open `about:debugging`, choose Debug for Brief, tell Brief to update feeds and check whether there are any errors in the console?

I think I have figured something out, this behavior only happens when I clear the cache/private data and restart the browser. When I do that, Brief asks me for storage permissions again upon restart, it seems that it forgets to permanently hold that permission. This is when I have to disable/restart/enable Brief to get it to work again. 

This makes me believe that there's an issue with the way that persistent storage permissions are working with the extension/Firefox.

**Comment by LaurentDardenne** [comment]
Hi,
same problem (and the feeds can not be deleted).
Brief : 2.6.0
Firefox : Version 68.0.2, August 14, 2019
Windows 10 1903

**Comment by jeremiah-rs** [comment]
> 
> 
> Hi,
> same problem (and the feeds can not be deleted).
> Brief : 2.6.0
> Firefox : Version 68.0.2, August 14, 2019
> Windows 10 1903

Yeah, it's still still there. I have gotten around it by not clearing the cache/private data and just using ccleaner to clear my cookies/cache.
